install-wings.sh

```diff
@@ -558,6 +634,29 @@ main() {
   $ASK_MYSQL && read -r CONFIRM_INSTALL_MARIADB
   $ASK_MYSQL && [[ "$CONFIRM_INSTALL_MARIADB" =~ [Yy] ]] && INSTALL_MARIADB=true
```

Comment:
Is it necessary to first ask if the user wants MariaDB installed, and then asking if they want the automatic setup? It seems useless to want the script to install MariaDB, but not do anything about it. What do you reckon?

Reply:
I think it is necessary because why would you want to automatically configure database hosts if you don't want MariaDB installed?

> It seems useless to want the script to install MariaDB, but not do anything about it.

Yeah, it is useless. But I'm still letting the user decide if they want it to be configured or not.

Reply:
Sorry, I realised I might have sounded a bit confused. I suggest just simply asking: do you want to automatically configure a database user? If the server has MariaDB, then just create the user. If the server does not have MariaDB, then install it and create the user. 🙂

Reply:
Do you reckon that’s a good solution?

Reply:
I like both solutions, mine and yours.

Reply:
Would there be a reason for someone to want MariaDB just installed and nothing else?

Reply:
Well, actually, if I think about it: No. I'll implement your solution

Reply:
Implemented.

Reply:
And I forgot to sign my commit again :^)